### PR TITLE
update AgentLoader script and document

### LINKED ISSUE #365: Optimize AgentLoader script
### What would you like to be added?

Optimize AgentLoader script

### Why is this needed?

Optimize AgentLoader script

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

Doc

What this PR does / why we need it?

update AgentLoader script

Which issue(s) this PR fixes:

Fixes #365

### CODE REVIEW

File: docs/zh/document/user-guide/sermant-agent.md [hunk 1]
```diff
@@ -435,9 +444,38 @@ public class AgentLoader {
...
+        for (int i = 0; i < fullCommand.size(); i++) {
+            System.out.println(i + ": " + fullCommand.get(i));
+        }
+        System.out.print("请输入您要执行命令的序号：");
+        int selectedCommandIndex = Integer.parseInt(userInputReader.readLine());
+
+        if (selectedCommandIndex < 0 || selectedCommandIndex >= fullCommand.size()) {
+            System.out.println("无效的命令序号");
+            return;
+        }
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: support choose command for three times

File: docs/zh/document/user-guide/sermant-agent.md [hunk 2]
```diff
@@ -435,9 +444,38 @@ public class AgentLoader {
         System.out.print("请输入Sermant Agent所在目录（默认采用该目录下sermant-agent.jar为入口）：");
         String agentPath = userInputReader.readLine();
 
+        // 展示目前支持的命令列表
+        System.out.println("请选择需要执行的命令（热插拔插件的命令当检测到Sermant Agent未安装时会自动安装Agent） :");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: description better place at every command

File: docs/zh/document/user-guide/sermant-agent.md [hunk 3]
```diff
@@ -435,9 +444,38 @@ public class AgentLoader {
...
+
+        if (selectedCommandIndex < 0 || selectedCommandIndex >= fullCommand.size()) {
+            System.out.println("无效的命令序号");
+            return;
+        }
+
+        String currentCommand = fullCommand.get(selectedCommandIndex);
+
+        if (pluginCommand.contains(currentCommand)) {
+            System.out.print("请输入您要执行的插件名称，多个插件使用/分隔：");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: 执行->操作

Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: doen

File: docs/zh/document/user-guide/sermant-agent.md (hunk 2 above)
Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

File: docs/zh/document/user-guide/sermant-agent.md (hunk 1 above)
Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

File: docs/zh/document/user-guide/sermant-agent.md [hunk 4]
```diff
@@ -446,6 +518,29 @@ public class AgentLoader {
...
+        FULL_COMMAND.add("INSTALL-AGENT");
+        FULL_COMMAND.add("UNINSTALL-AGENT");
+        FULL_COMMAND.add("INSTALL-PLUGINS");
+        FULL_COMMAND.add("UNINSTALL-PLUGINS");
+        FULL_COMMAND.add("UPDATE-PLUGINS");
+        FULL_COMMAND.add("CHECK-ENHANCEMENT");
+
+        // 命令描述
+        COMMAND_DETAILS.put("INSTALL-AGENT", "安装Sermant Agent");
+        COMMAND_DETAILS.put("UNINSTALL-AGENT", "卸载Sermant Agent");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: 所有已安装插件都会被卸载

File: docs/zh/document/user-guide/sermant-agent.md [hunk 5]
```diff
@@ -446,6 +518,29 @@ public class AgentLoader {
...
+        // 填充目前支持的命令
+        FULL_COMMAND.add("INSTALL-AGENT");
+        FULL_COMMAND.add("UNINSTALL-AGENT");
+        FULL_COMMAND.add("INSTALL-PLUGINS");
+        FULL_COMMAND.add("UNINSTALL-PLUGINS");
+        FULL_COMMAND.add("UPDATE-PLUGINS");
+        FULL_COMMAND.add("CHECK-ENHANCEMENT");
+
+        // 命令描述
+        COMMAND_DETAILS.put("INSTALL-AGENT", "安装Sermant Agent");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: 启动时会挂哪些插件

File: docs/zh/document/user-guide/sermant-agent.md [hunk 6]
```diff
@@ -446,6 +518,29 @@ public class AgentLoader {
...
+        COMMAND_DETAILS.put("UNINSTALL-AGENT", "卸载Sermant Agent");
+        COMMAND_DETAILS.put("INSTALL-PLUGINS", "安装Sermant Agent插件，Sermant Agent未安装时会自动安装Agent");
+        COMMAND_DETAILS.put("UNINSTALL-PLUGINS", "卸载Sermant Agent插件");
+        COMMAND_DETAILS.put("UPDATE-PLUGINS", "更新Sermant Agent插件");
+        COMMAND_DETAILS.put("CHECK-ENHANCEMENT", "查询增强信息");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: 描述清楚具体信息

File: docs/zh/document/user-guide/sermant-agent.md (hunk 4 above)
Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

File: docs/zh/document/user-guide/sermant-agent.md (hunk 5 above)
Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

File: docs/zh/document/user-guide/sermant-agent.md (hunk 6 above)
Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

File: docs/zh/document/user-guide/sermant-agent.md [hunk 7]
```diff
@@ -435,16 +535,99 @@ public class AgentLoader {
...
+        FULL_COMMAND.add("UPDATE-PLUGINS");
+        FULL_COMMAND.add("CHECK-ENHANCEMENT");
+
+        // 命令描述
+        COMMAND_DETAILS.put("INSTALL-AGENT", "安装Sermant Agent，同时安装plugins.yaml配置文件中dynamicPlugins.active下的所有插件");
+        COMMAND_DETAILS.put("UNINSTALL-AGENT", "卸载Sermant Agent，同时卸载所有已安装插件");
+        COMMAND_DETAILS.put("INSTALL-PLUGINS", "安装插件至Sermant Agent中，Sermant Agent未安装时会自动安装Agent（同时安装plugins"
+                + ".yaml配置文件中dynamicPlugins.active下的所有插件）");
+        COMMAND_DETAILS.put("UNINSTALL-PLUGINS", "卸载Sermant Agent中的插件");
+        COMMAND_DETAILS.put("UPDATE-PLUGINS", "更新Sermant Agent插件");
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-agent.md]: 卸载Sermant Agent中已安装的插件
更新Sermant Agent中已安装的插件

Comment (daizhenyu) [reply docs/zh/document/user-guide/sermant-agent.md]: done

**Review by lilai23 — APPROVED** [review]

**Pull request merged**